illumina_uploader/illumina_uploader.py

```diff
@@ -47,8 +49,8 @@ def main():
         sequencer = localInfo["sequencer"]
     else:
         sequencer = args.sequencer
-    inputDirs = localInfo["inputDirs"].split(",")
```

Comment:
I believe this was a typo. Should probably adopt python style guide https://peps.python.org/pep-0008/